src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+* {
+  margin: 0;
```

Comment:
Only `box-sizing` allowed for this selector 
![image](https://user-images.githubusercontent.com/70009205/222507356-46a6ff3f-e6ea-4cf7-af58-ed3308a21415.png)

Reply:
how delete all margin and padding without *selector&

Reply:
reset them only for the elements which need it. you can do it in class selectors or use a reset file, to reset them for specific tags. And better ask questions in the chat, you will get the response much faster